templates/api/keras_cv/models/index.md

```diff
@@ -0,0 +1,88 @@
+# KerasCV Models
+
+KerasCV contains end-to-end implementations of popular model
+architectures. These models can be created in two ways:
+
+- Through the `from_preset()` constructor, which instantiates an object with
+  a pre-trained configurations, and (optionally) weights.
+  Available preset names are listed on this page.
+
+```python
+model = keras_cv.models.RetinaNet.from_preset(
+    "resnet50_v2_imagenet",
+    num_classes=20,
+    bounding_box_format="xywh",
+)
+```
+
+- Through custom configuration controlled by the user. To do this, simply
+  pass the desired configuration parameters to the default constructors of the
+  symbols documented below.
+
+```python
+backbone = keras_cv.models.ResNetBackbone(
+    stackwise_filters=[64, 128, 256, 512],
+    stackwise_blocks=[2, 2, 2, 2],
+    stackwise_strides=[1, 2, 2, 2],
+    include_rescaling=False,
+)
+model = keras_cv.models.RetinaNet(
+    backbone=backbone,
+    num_classes=20,
+    bounding_box_format="xywh",
+)
+```
+
+For a more in depth introduction to how our API fits together, see the
+[getting started guide](guides/keras_cv/).
+
+## Backbone presets
+
+The following preset names correspond to a configuration, and weights
+for a **backbone**. These presets are not inference-ready, and must be
+fine-tuned for a given task!
+
+The names below can be used with the `from_preset()` constructor for the
+corresponding **backbone** model.
+
+```python
+backbone = keras_cv.models.ResNetBackbone.from_preset("resnet50_imagenet")
+```
+**Note**: All pretrained weights should be used with image input data in range
```

Comment:
Need to discuss the interaction with `include_rescaling`. Here's a version with more color:

**Note**: All pretrained weights should be used with unnormalized pixel intensities in the range `[0, 255]` if `include_rescaling=True` or in the range `[0, 1]` if `including_rescaling=False`.

Reply:
Done.